doc/man3/OCSP_sendreq_new.pod

```diff
@@ -53,6 +53,11 @@ L<openssl_user_macros(7)>:
 
 =head1 DESCRIPTION
 
+These functions only perform a minimal HTTP query to a responder.  For
+example, they only support HTTP POST queries to OCSP responders.  If an
+application needs more advanced features it should use a more complete
+HTTP library.
```

Comment:
I think that meanwhile the sentence 
"If an application needs more advanced features it should use a more complete HTTP library." 
is not to-the-point anymore. The HTTP library has become more capable; in particular it does support GET queries. So I suggest dropping that sentence.

The two sentences before are still valid, though. 
I just ask to place each of them in a separate line since I learned some time ago that in POD files there should not be end of sentences in the middle of a line.